### PR TITLE
implement the functionality that allows a user get all orders

### PR DESCRIPTION
**Pull request by andrewinsoul**
### WHAT DOES THIS PR DO?
Have the get all orders endpoint working in Fast Food REST API

### DESCRIPTION OF TASK TO BE COMPLETED
- Set up express server
- Have GET /orders working

### HOW SHOULD THIS BE MANUALLY TESTED?
After cloning the repo, cd into it and RUN npm start, using postman, test the endpoint GET /orders
It should return status 200 with an empty array as no orders are yet added.

###WHAT ARE THE RELEVANT PIVOTAL TRACKER STORIES?
#160539515

### CODE REVIEW

**Comment by dogoyaro** [comment]
The description of tasks to be completed should be a description of the tasks you took to "Have GET/ orders working". What steps did you take to get it working?

The headers in your PR description, e.g "Description of task to be completed" doesn't have to be in all caps.

Check that the markdown is appropriately formatted.

**Comment by dogoyaro** [comment]
Your commit message doesn't follow the prescribed format. Please check [this](https://github.com/andela/project-jiayou-cfh/wiki/Git-Branch-Naming-Convention), and correct accordingly

File: server/models/order.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const orders = [];
+export default orders;
```
Comment (dogoyaro) [inline server/models/order.js:2]: The default export should be the same name as the file. Change `order.js` to `orders.js`.

File: server/controllers/order.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import order from '../models/order';
+
+class OrderClass {
```
Comment (dogoyaro) [inline server/controllers/order.js]: `class Order` is a superior name to `class OrderClass`

File: server/models/order.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+const orders = [];
```
Comment (dogoyaro) [inline server/models/order.js:1]: Are you certain that an array is the best data structure to store orders?

File: .eslintrc.js [hunk 4]
```diff
@@ -1,3 +1,8 @@
 module.exports = {
-    "extends": "airbnb-base"
+    "extends": "airbnb-base",
+    "rules": {
+        "linebreak-style": 0,
+        "class-methods-use-this": 0,
+        "no-console": 0,
+    }
 };
```
Comment (dogoyaro) [inline .eslintrc.js]: I think you need a newline at the end of this file.

**Review by dogoyaro — CHANGES_REQUESTED** [review]

File: server/models/order.js (hunk 3 above)
Comment (andrewinsoul) [reply server/models/order.js:1]: Yes, it has method that allows for easy retrieval of an item and also update an item. Methods like push, findIndex, find and other array methods makes me think it's the suitable data structure for the job

**Comment by andrewinsoul** [comment]
feedback implemented

File: server/models/order.js (hunk 1 above)
Comment (dogoyaro) [reply server/models/order.js:2]: @andrewinsoul. Please address this issue so we can merge this PR.

**Pull request merged**